### PR TITLE
[14.0][MIG] sale_commission: Migration to 14.0

### PR DESCRIPTION
**Pull request by fredzamoabg**
sale_commission module migration from 13.0
(alignment of https://github.com/OCA/commission/pull/273)

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: sale_commission/wizard/wizard_settle.py [hunk 1]
```diff
@@ -114,6 +114,7 @@ def action_settle(self):
                                 )
                             )
                         settlement_ids.append(settlement.id)
+                    line.read(["amount"])  # force amount computation
```
Comment (pedrobaeza) [inline sale_commission/wizard/wizard_settle.py]: This seems weird and shouldn't be needed. What are the steps for needing this?

Comment (fredzamoabg) [reply sale_commission/wizard/wizard_settle.py]: You are right, but running tests was found error related amount computation that causing exception per validation in **_check_settle_integrity** method. I'll try to find out the reason in order to remove that line.

thanks

Comment (fredzamoabg) [reply sale_commission/wizard/wizard_settle.py]: removed and added amount field computation in tests

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**